### ISSUE TITLE
Example of js_export use

### ISSUE THREAD

**Issue by David-OConnor** [koute/stdweb#244]
I think the documentation on the main page needs a full example of js_export use that includes how to import it, or one of the examples in the examples folder should include this. 

When I try #[macro_use]:
```error: attribute procedural macros cannot be imported with `#[macro_use]`
...
help: instead, import the procedural macro like any other item: `use stdweb_internal_macros::js_export;`
```

If I try to import stdweb::stdweb_internal_macros::js_export:
`module `stdweb_internal_macros` is private`

**Comment by koute** [comment]
Yes, wee already have an example crate for that which is linked to in the README. (:

https://github.com/koute/stdweb/tree/master/examples/hasher
